creator-node/src/services/sync/syncImmediateQueue.js

```diff
@@ -90,6 +96,17 @@ class SyncImmediateQueue {
     }
   }
 
+  async deleteOldActiveJobs() {
+    const oldActiveJobs = await this.queue.getJobs(['active'])
+    logger.info(
```

Comment:
I know we're trying to remove logs but this one seemed important enough and infrequent enough to keep as `info`